docs/wiki/99 - Developers/SQL Dictionary.md

```diff
@@ -47,3 +47,37 @@
 	| `WindowMidpointDays `| Candidate's ideal age in days for a visit                                                                 |  |
 	
+
+- Table: `Project`
+ 
+ 	*This table stores the list of projects configured for the study.*
+ 
+	|         Field        |                  Description                  |                                        Notes                                                 |
+	|:--------------------:|:---------------------------------------------:|:--------------------------------------------------------------------------------------------:|
+	| `ProjectID`          | Identifier of the project                     | Avoid setting this field explicitly when inserting data, it auto increments.                 |
+	| `Name`               | Full name of the project                      |                                                                                              |
+	| `recruitmentTarget`  | Expected number of candidates to be recruited |                                                                                              |
+
+
+- Table: `subproject`
+ 
+ 	*This table stores the list of subprojects configured for the study.*
+ 
+	|        Field       |                  Description                  |                                     Notes                                    |
+	|:------------------:|:---------------------------------------------:|:----------------------------------------------------------------------------:|
+	| `SubprojectID`     | Identifier of the subproject                  | Avoid setting this field explicitly when inserting data, it auto increments. |
+	| `title`            | Name of the subproject                        |                                                                              |
+	| `useEDC`           | Use the Expected date Of Confinement          |                                                                              |
+	| `WindowDifference` |                                               | Deprecated ??                                                                |
+	| `RecruitmentTarget`| Expected number of candidates to be recruited |                                                                              |
+	
+	
+- Table: `project_subproject_rel`
+ 
+ 	*This table stores the association of projects with subprojects*
+ 
+	|            Field          |                    Description                   |                                     Notes                                    |
+	|:-------------------------:|:------------------------------------------------:|:----------------------------------------------------------------------------:|
+	| `ProjectSubprojectRelID ` | Identifier of the project-subproject relation    | Avoid setting this field explicitly when inserting data, it auto increments. |
```

Comment:
Why does this field exist for a rel table? It should probably be explained in the notes.

Reply:
it's to be able to point to a single row from another rel table
the visit_project_subproject_rel

Reply:
just a note that other tables need it as a foreign key would be enough, I think.